shell/sct_run_batch.sh

```diff
@@ -53,26 +53,48 @@ command_open() {
   fi
 }
 
+# Usage of this script
+usage() {
+  echo -e "Correct usage ./sct_run_batch.sh <parameters.sh> <process_data.sh>"
+  echo -e "-s  processing subjects sequentially"
```

Comment:
i would make the usage more explicit:
~~~
  echo -e "-s  Processing subjects sequentially, without parallel processing."
~~~
Also, I would ass a description of the function. Something like:
~~~
This function runs a batch script across multiple subjects that are present in a source folder (as defined in the file parameters.sh). If GNU parallel is installed, it will use it by distributing subjects across the available CPU cores. For more information about the file parameters.sh, see the example file: $SCT_DIR/shell/parameters_example.sh.
~~~